woocommerce/woocommerce
# In-app marketplace style tweaks

### All Submissions:


### Changes proposed in this Pull Request:

Minor style tweaks to make sure the in-app marketplace matches the [design](seb2o1DExPxoh5d53ftJif-fi-812%3A1).

* Add top margin above `.addons-button` in promos, to ensure there's a proper gap above it on smaller viewports.
* Slightly change the wording and styling of the `Search results for` label at the top of extensions search results.

### How to test the changes in this Pull Request:

1. Check out the branch.
2. Make sure your store is set to a US location in `/wp-admin/admin.php?page=wc-settings`, and WooCommerce Payments is not active in your plugins.
3. Go to `wp-admin/admin.php?page=wc-addons&section=payment-gateways`. Check that the WCPay promo looks as illustrated in the screenshot below.
4. Do a search in the search input on the Marketplace page, or go to `wp-admin/admin.php?search=shipping&page=wc-addons&section=_all`.
5. Check that the `Search results for` label matches the designs.

### Notes

You can ignore the "changes" the GitHub diff is showing in files other than the first two, `assets/css/admin.scss` and `includes/admin/views/html-admin-page-addons.php`. After an update of this branch, it's highlighting "changes" that already exist in `trunk` and `update/marketplace-1`.

### Screenshots

#### WCPay promo

<img width="500" alt="image" src="https://user-images.githubusercontent.com/1647564/129334088-7e2e5359-e28a-43da-b088-f69cd5be05b3.png">

#### Search results

<img width="500" alt="image" src="https://user-images.githubusercontent.com/1647564/129334192-2a6aa736-1cad-46c3-968e-18cadee5fce2.png">

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### FOR PR REVIEWER ONLY:
